config/locales/provider_interface/candidate_pool_invites.yml

```diff
@@ -10,6 +10,7 @@ en:
         index:
           title: Find candidates - Invited
           heading: Find candidates
+          table_caption: Find candidates
```

Comment:
I don't think this is necessary, the invited index calls a component to render the table and it already has a caption.